add Feature

Sbaah opened the pull request:
Since the Minify image was done, I decide to add Responsive Web Design for the image to it.

Sbaah committed:
add Feature

viniciuskneves commented on index.html:
```diff
@@ -3,6 +3,7 @@
 <head>
   <!-- Global site tag (gtag.js) - Google Analytics -->
   <script async src="https://www.googletagmanager.com/gtag/js?id=UA-148502409-1"></script>
+  <script type="text/javascript" src="http://code.jquery.com/jquery-latest.min.js"></script>
```
Hey, what is the reason behind `jQuery` here?

Sbaah replied:
I was test some code but i will remove it.

viniciuskneves commented on package.json:
```diff
@@ -19,6 +19,7 @@
   },
   "homepage": "https://ichbineinekartoffel.de",
   "devDependencies": {
+    "cssnano": "^4.1.10",
```
This guy as well, what is the reason behind it?

Sbaah replied:
When installed Parcel with was added to the package.

Sbaah committed:
Updated For Pull Request

Removed the  un-used jquery from the index.html

Removed " "cssnano": "^4.1.10", " from the package file.
